lib/scala/kg/src/main/scala/io/github/tetherlessworld/mcsapps/lib/kg/stores/postgres/PostgresKgCommandStore.scala

```diff
@@ -11,22 +11,23 @@ import play.api.db.slick.DatabaseConfigProvider
 
 import scala.concurrent.ExecutionContext
 
+@Singleton
```

Comment:
Worth reading up on Guice if you haven't. It's a good example of Dependency Injection frameworks, which are a useful thing to have in your toolkit.

Reply:
Yep am trying that out for the PostgresConfigProvider